sdk/notificationhubs/notification-hubs/src/client/internal/_client.ts

```diff
@@ -38,7 +36,9 @@ export function createRequest(
 }
 
 /**
- * @internal
```

Comment:
isn't this still an internal export, so shouldn't it still be tagged as internal?

Reply:
This file isn't exported externally nor is anything from here re-exported.